fnet/network.go

```diff
@@ -160,3 +160,20 @@ func Proxy(port string, dest *net.TCPAddr) *net.TCPAddr {
 func ProxyToDestination(listenPort string, destination string) *net.TCPAddr {
 	return Proxy(listenPort, ResolveDestination(destination))
 }
+
+// Join the input Host And Port in the form of hostname:port, ip:port or :port. And returns
+func JoinHostAndPort(inputPort string, addr *net.TCPAddr) string {
+	urlHostPort := inputPort
+	portStr := ""
```

Comment:
this used to be
```
portStr := inputPort
```
in the original code, I doubt it works the same without it ?

Reply:
I used it for different purpose. portStr indicates the port that gathered from net.TCPAddr